bin/GameIndex.dbf

```diff
@@ -1,5 +1,5 @@
 ---------------------------------------------
--- PCSX2 Game DataBase!
+-- PCSX2 Game Database!
```

Comment:
I prefer old.

Reply:
Wiki says Database, I agree with that tbh.

Reply:
Correct use is Database and it fits better is the reason i changed it.